Source/WebKit/UIProcess/Cocoa/WKTextExtractionItem.swift

```diff
@@ -23,8 +23,7 @@
 
 import Foundation
 
-@available(iOS 17.0, macOS 12.0, *)
-@objc(WKTextExtractionItem) public class WKTextExtractionItem: NSObject {
+@objc(WKTextExtractionItem) class WKTextExtractionItem: NSObject {
```

Comment:
This appears to be implementing an `@interface` from a `.h` file? If so, you should really consider using `@_objcImplementation` instead. While it is possible to alias the header definition like this, you get no guarantees from the compiler and no help ensuring you've implemented things correctly.

https://github.com/swiftlang/swift-evolution/blob/main/proposals/0436-objc-implementation.md